feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts

```diff
@@ -63,7 +63,7 @@ export class ConfiguratorFormComponent implements OnInit {
             .hasConflicts(routingData.owner)
             .pipe(take(1))
             .subscribe((hasConflicts) => {
-              if (hasConflicts) {
+              if (hasConflicts && !routingData.ignoreConflicts) {
```

Comment:
Change comment in else statement in line 71.
"//Only check for Incomplete group when there are no conflicts or conflicts should be ignored"